### PR TITLE
Modify to fit Vaana setup

### PR DESCRIPTION
**Pull request by uhef**
This has not been successfully ran on a master account yet.

### CODE REVIEW

File: accounts/infosec/users.tf [hunk 1]
```diff
@@ -1,33 +1,34 @@
+# Skip creation of users. They will be created manually afterwards
```
Comment (jackap) [inline accounts/infosec/users.tf]: I would still rely on terraform/terragrunt for user creation

File: README.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # AWS Organization Terraform
 
+This is a fork of Liatrio secure AWS account setup project with following modifications:
+
+- Adds way to run the scripts without local installation of Terraform or Terragrunt by providing a way to run these in a docker container.
+- Replaced dependency to [Keybase](https://keybase.io) with plain gpg2 usage.
+- Remove possibility to create one-time logins for users as user account creation will be handled manually afterwards.
```
Comment (jackap) [inline README.md]: See comment on `accounts/infosec/user.tf`

File: master/init.sh [hunk 3]
```diff
@@ -9,12 +9,11 @@ function usage {
     echo "  *** MUST BE RUN WITH ADMIN CREDENTIALS FOR terraform-init USER IN THE MASTER ACCOUNT ***"
     echo ""
     echo "USAGE:"
-    echo "  init.sh -a terraform_init_access_key -s terraform_init_secret_key -k keybase_profile"
+    echo "  init.sh -a terraform_init_access_key -s terraform_init_secret_key -p pgp_key_name"
```
Comment (jackap) [inline master/init.sh]: maybe we can still use `-k` as we are specifying a key

File: master/init.sh [hunk 4]
```diff
@@ -123,40 +129,29 @@ terragrunt init
...
+unset "TF_VAR_pgp_public_key"
 popd
 
 echo "=== INITIALIZATION COMPLETE ==="
 
-if [[ -n "${LOGIN_USER}" ]]; then
-    echo "One-time password for ${LOGIN_USER}: $(echo ${ENCRYPTED_PASS} | base64 --decode | keybase pgp decrypt)"
-fi
-
 echo "Login URL: https://${INFOSEC_ALIAS}.signin.aws.amazon.com/console"
```
Comment (jackap) [inline master/init.sh]: We can remove all lines below.  We are not creating any user with the current modifications

**Comment by uhef** [comment]
> Some things I realised while reading the code:
> 
> * Where are the s3 buckets for the newly created accounts?

They don't get created here. This only creates the account structure, user groups, policies, roles and cloudtrails. After this we need to bootstrap the new accounts. The boostrapping could be part of `vaana-infra`.

> * Each infosec user has also access to billing information

There's a separate user group `MasterBilling`. User has to be included in this group in order to access billing information.

> * Do we want to use staging and production admins? If so we must be aware that each of these users has privileges (e.g., create accounts).

We need to have staging and production admins. I think developers group have restricted access. We might have to revisit the policy attached to developer role in staging and production account and increase the rights in order for all developers to be able to run terraform and create new resources, etc.

> 
> Also take a look at [liatrio#7](https://github.com/liatrio/aws-accounts-terraform/issues/7)

I think the terraform state bucket gets created in the master account. This is alright for us.

Role assumptions is a lot to keep track, as the guy says. I hope I'm right in simplifying it to following sentences, though: *All users and user groups are defined in infosec account. When users are attached to groups they get rights specified by AWS policies. AWS policy gives a user a right to assume a role in another account. This account can be staging or production. There are roles Developer and Administrator for both staging and production accounts. When a user assumes this role they can perform actions in staging or production accounts.*

What happens after this is up to us. Next step would probably be to setup staging and production accounts by bootstrapping them and running vaana-infra on them. This although requires us to solve a few new challenges such as how do we manage Route 53 hosted zones when everything is under vaana.fi.

Developer with administrator access to staging can make changes to staging account. Whether this repo should be used for that is another question. We probably want to do rest in some other repo, like `vaana-infra`

One can extend this repo to add a functionality to add more accounts in the future. Given this it should not be a big effort. There needs to be a new entry-point script aside `init.sh` to achieve this. Perhaps something like `introduce-account.sh`.

File: accounts/infosec/users.tf (hunk 1 above)
Comment (uhef) [reply accounts/infosec/users.tf]: We can discuss this later once we have ran the initialization.

File: master/init.sh (hunk 3 above)
Comment (uhef) [reply master/init.sh]: I changed the parameter name in order not to confuse it with the keybase profile parameter. Now its explicit that we anticipate something else than keybase profile name.

File: master/init.sh (hunk 4 above)
Comment (uhef) [reply master/init.sh]: The lines below are not about users. They are about roles. I know the terminology is hard to grasp but one shouldn't confuse user and role with each other. I think the URLs below give you a way to _swich between roles_, as they say. As we know, there's a bunch of assume role policies created when we run this script. This means that a user X that can assume role A or role B can switch between those roles. If you have logged into AWS as this user X you can use the URLs below to switch between roles.

**Pull request merged**